kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -125,9 +139,14 @@ download_akse()
         exit 1
     fi
     
-    TEMPLATE_URL="https://raw.githubusercontent.com/$GALLERY_REPO/$GALLERY_BRANCH/kubernetes/template/DeploymentTemplates/$DEFINITION_TEMPLATE_NAME"
-    curl --retry 5 --retry-delay 10 --max-time 60 -s -f -O $TEMPLATE_URL || exit $ERR_TEMPLATE_DOWNLOAD
-
+    if [ ! $DISCONNECTED_AKS_ENGINE_URL ]
```

Comment:
do we want to keep the code that pulls down the api-template? I would be nice to remove that dependency